projects/js-packages/publicize-components/src/components/media-section-v2/index.tsx

```diff
@@ -68,7 +68,7 @@ export default function MediaSectionV2( {
 		if ( currentSource === 'featured-image' ) {
```

Comment:
In some follow-up, I think we can break this file into smaller components with an improved API for the pieces. I may do so in the coming days.